### DISCUSSION TITLE
False positives for sentences ending in `:`

### DISCUSSION THREAD

**Discussion by bkhl** [minad/jinx#196] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/c77b4c83-0545-4876-a948-46e8c60963e0)

I have an issue where any words at the end of a sentence ending in `:` gets highlighted as an error.

Anyone have an idea on how to configure Jinx (or the underlying library) to correctly tokenize these words?

**Comment by minad** [comment, marked as answer]
I would expect that `:` is not a word character. However it seems that the underlying dictionary reports it as such. You can override this by modifying `jinx--syntax-table` after loading the dictionary.

~~~ elisp
(modify-syntax-entry ?: "." jinx--syntax-table)
~~~

EDIT: I've added `jinx--syntax-overrides` in https://github.com/minad/jinx/commit/967f70115fb286fd0bc0f5634ae7d3d007a8956a to make such overrides a little easier.
